kubeflow/notebooks/prototypes/README.md

```diff
@@ -0,0 +1,131 @@
+## Goals
+
+- provide a k8 native mechanism to spawning jupyter notebooks for users
+
+- spawn a notebook within a RBAC protected namespace for a user
+
+- set a serviceAccountName within the spawning pod similar to jupyter-notebook
```

Comment:
nit: point (2), (3), (4) doesn't really seem related to this CRD. I feel like the namespace RBAC, service account attachment, and k8s auth is really external to this PR. 

The same CRD should also work on a k8s cluster if I use a `cluster-admin` role to create a `namespace` and launch it right?

This CRD feels like it is more wrapping the logic for handling networking and maybe populating jupyter specific default command, args, and env vars in pod templates.

Reply:
Thanks Jason, this is a nice summary and should be exactly what the notebooks component provides. I'll revisit the documentation and any assumptions I've made in the code.